pipelines/event-replay-pipeline/Pipfile

```diff
@@ -17,6 +17,11 @@ awscli = "*"
 pytest = "*"
 pytest-mock = "*"
 pytest-cov = "*"
+werkzeug = ">=0.15.3"
```

Comment:
The Pipfile.lock files need to be updated as well

Reply:
they have been updated